csource/common.go

```diff
@@ -203,6 +225,8 @@ static void execute_command(const char* format, ...)
 
 int tunfd = -1;
 
+#define TUN_MTU 1500
```

Comment:
Yes, jumbo frames is what I have in mind. I've seen some numbers as high as 64K somewhere, but don't remember where.
What will happen if fuzzer increases MTU to 9000 and sends a 8K packet and we read only 1500? WIll it be truncated? Split?